src/test/java/edu/kis/powp/jobs2d/events/SelectClearMacro.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.events;
+
+
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.drivers.MacroDriver;
+import edu.kis.powp.jobs2d.features.MacroFeature;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectClearMacro implements ActionListener {
+    private DriverManager driverManager;
+    public SelectClearMacro(DriverManager driverManager){
+        this.driverManager=driverManager;
+    }
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        MacroDriver manager = MacroFeature.getMacroDriver();
```

Comment:
Have you test sequence: "LoadMacro", "ClearMacro", "RunCommand"? It may not work correctly.